demos/tests/run_tests.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+Test script for the demos.
+
+The test data directory must contain a subdirectory "ILSVRC2012_img_val"
+containing the ILSVRC2012 dataset.
+"""
+
+import argparse
+import collections
+import itertools
+import json
+import shlex
+import shutil
+import subprocess
+import sys
+import tempfile
+
+from pathlib import Path
+
+from args import ArgContext
+from cases import DEMOS
+from image_sequences import IMAGE_SEQUENCES
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--demo-build-dir', type=Path, required=True)
+    parser.add_argument('--test-data-dir', type=Path, required=True)
+    parser.add_argument('--downloader-cache-dir', type=Path, required=True)
+    parser.add_argument('--demos')
+    return parser.parse_args()
+
+def main():
+    args = parse_args()
+
+    omz_dir = (Path(__file__).parent / '../..').resolve()
```

Comment:
> Are tests going to be supported for Windows?

Yes, eventually.

> In this case it can be a bad idea to use / in ../...

No, because `pathlib` automatically converts slashes to the appropriate directory separator (and even if it didn't, Windows is fine with forward slashes).

Reply:
You are right.